pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -77,7 +79,11 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional (postgresql != null) postgresql;
 
   nativeBuildInputs = [ bison flex gperf lndir perl pkg-config which ]
-    ++ lib.optionals stdenv.isDarwin [ xcbuild ];
+    ++ lib.optionals stdenv.isDarwin [ xcbuild ]
+    # `qtbase` expects to find `cc` (with no prefix) in the
```

Comment:
Can we patch that?

Why do we need this in the wrapper and package?

Reply:
I agree, there's a lot of ugliness in nixpkgs' `qt5` expressions.  But I am trying really hard not to undertake a rewrite-from-scratch of them here.